packages/ui/src/components/va-toast/VaToast.stories.ts

```diff
@@ -1,12 +1,28 @@
 import { defineComponent } from 'vue'
-import VaToast from './VaToast.demo.vue'
+import VaToastDemo from './VaToast.demo.vue'
+import VaToast from './VaToast.vue'
+import { StoryFn } from '@storybook/vue3'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaToast',
   component: VaToast,
 }
 
-export const Default = defineComponent({
-  components: { VaToast },
-  template: '<VaToast/>',
+export const Default = () => defineComponent({
+  components: { VaToastDemo },
+  template: '<VaToastDemo/>',
 })
+
+export const TextColor: StoryFn = () => ({
+  components: { VaToast: VaToast },
+  template: '<VaToast color="red"/>',
```

Comment:
@asvae

What should I do to test `<VaToast color="black" />` and `<VaToast color="white" />`?

Reply:
```
Color

[black]
VaToast color=black

[white]
VaToast color=white
```

Do we need this test though? I though it used general logic.

If yes you might also want to have no color as an option, i.e.

[primary?]
VaToast

[black]
VaToast color=black

[white]
VaToast color=white

Reply:
Fixed.